### ISSUE TITLE
Documentation for v1.3.1

### ISSUE THREAD

**Issue by PriyaDoIT** [Chicago/opengrid#285]
We need to update the documentation in the docs folder for the new configuration options in milestone 1.3.1.    Once this is complete, @nfspeedypur to release this milestone in production.

**Comment by rladines** [comment]
Now in DEV, see links below.

http://opengrid.readthedocs.io/en/dev/OpenGrid%20API/
http://opengrid.readthedocs.io/en/dev/OpenGrid%20Configuration/

**Comment by PriyaDoIT** [comment]
duplicate of #286

**Comment by PriyaDoIT** [comment]
Documentation is finished and closed in #286.

@nfspeedypur to release this milestone to chicago.opengrid.io
